ksqldb-common/src/main/java/io/confluent/ksql/util/ConsistencyOffsetVector.java

```diff
@@ -214,8 +223,14 @@ public boolean equals(final Object o) {
     if (o == null || getClass() != o.getClass()) {
       return false;
     }
-    final ConsistencyOffsetVector that = (ConsistencyOffsetVector) o;
-    return this.dominates(that) && that.dominates(this);
```

Comment:
Using `dominates` is wrong as it doesn't not account for missing topics/partitions